src/Serilog.Sinks.AzureTableStorage/Sinks/AzureTableStorage/AzureBatchingTableStorageSink.cs

```diff
@@ -51,7 +53,7 @@ public AzureBatchingTableStorageSink(CloudStorageAccount storageAccount, IFormat
             if (string.IsNullOrEmpty(storageTableName)) storageTableName = typeof(LogEventEntity).Name;
 
             _table = tableClient.GetTableReference(storageTableName);
-            _table.CreateIfNotExists();
+            _table.CreateIfNotExistsAsync().Wait(_waitTimeoutMilliseconds);
```

Comment:
It won't be a background call - nice catch @robdmoore.

Here's how we handle this issue elsewhere: https://github.com/serilog/serilog-sinks-periodicbatching/blob/dev/src/Serilog.Sinks.PeriodicBatching/Sinks/PeriodicBatching/PeriodicBatchingSink.cs#L118